/**:
  ros__parameters:
    camera_name: "camera_1"
    frame_id: "camera_top/camera_link"
    pixel_format: mono8
    #serial_no: 220300119
    serial_no: 233600049
    #camera_info_url: "file:///home/kaan/projects/autoware_arena_ws/src/lucid_vision_driver/config/test.yaml"
    camera_info_url: "/home/claydergc/ros2_ws/src/lucid_vision_driver/config/test_full_res.yaml"
    fps: 21
    horizontal_binning: 1 # resize horizontal resolution/horizontal_binning
    vertical_binning: 1 # resize horizontal resolution/vertical_binning
    enable_rectifying: false
    enable_compressing: false
    image_horizontal_flip: false
    image_vertical_flip: false
    use_default_device_settings: true
    # if you use_default_device_settings: true, the following settings are ignored.
    # ROS driver will use the default device settings of the camera.
    exposure_auto: true
    exposure_target: 187
    gain_auto: false
    gain_target: 1
    gamma_target: 1.0
    image_horizontal_flip: false
    image_vertical_flip: false


